### ISSUE TITLE
Questions an amateur

### ISSUE THREAD

**Issue by jimlynnjulian** [pymodbus-dev/pymodbus#2445]
I'm examining the use of PyModbus to communicate with a Epever Triron charge controller thaqt uses modbus RTU protocol.

   The latest version is 3.7.4, yet Pypi suggests 2.0.0.
   ???
   If I use 2.0.0, the documentation will not be an exact fit since changes will have been made between the two versions.
   Should I install the newer version?
   Any advice, comment, ideas?

Thanks.

**Comment by janiversen** [comment]
The are no v2.0.0

latest on pypi and github are v3.7.4

**Comment by jimlynnjulian** [comment]
https://pypi.org/project/pymodbus/2.0.0/

**Comment by MrWaloo** [comment]
I have found 1.0.0
https://pypi.org/project/pymodbus/1.0.0/

But still, the latest version is 3.7.4
https://pypi.org/project/pymodbus/

**Comment by janiversen** [comment]
if you look in menu, you cannot select versions earlier than 3.7.0,.

However old versions are kept forever on pypi and therefore you can get the old versions by building the url yourself. pypi only recommend the "latest stable" version in this case v3.7.4 so I am not sure why you think pypi recommends 2.0.0

**Comment by jimlynnjulian** [comment]
I found the link searching for modbus libraries with google. 
I found the newer verison searching within Pypi, after reading your posts. 
An assumption was made concerning the currency of the google search. My bad.

As for old versions, maybe a few requests regarding the issue, from users and especially posters, would change their practice. Perhaps an email to Google?

If, and when, I download pymodbus, I will, as always, peruse the README first.

Note: there are many computers from many years running software from many versions.
Hardware may necessitate recreating a software environment from the past for compatibility reasons.

An example is the usefulness of Python 2 to 3 conversion software and online converters and guides.
Some of the software I've written to run a solar power station was written for Python 2.7 and is permanently installed.
Rewrite the scripts for Python3? Sometime, maybe. 

But I've got a whole lot to do and time is at a premium. 
The rewrite is low on my list of priorities and I currently limit updates to those absolutely necessary.

Thank you for your time.
Have a nice day.
